src/borg/testsuite/archiver/rinfo_cmd.py

```diff
@@ -36,6 +37,13 @@ def test_info_json(self):
         assert all(isinstance(o, int) for o in stats.values())
         assert all(key in stats for key in ("total_chunks", "total_size", "total_unique_chunks", "unique_size"))
 
+    def test_info_on_repository_with_storage_quota(self):
+        self.create_regular_file("file1", size=1024 * 80)
+        self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION, "--storage-quota=1G")
+        self.cmd(f"--repo={self.repository_location}", "create", "test", "input")
+        info_repo = self.cmd(f"--repo={self.repository_location}", "rinfo")
+        assert "Storage quota" in info_repo
```

Comment:
would be good if this also tested the 2 values (used/total).

Reply:
I have a problem with this. In the test I added the used quota is different if I execute the test file or if I execute the test on its own.
The difference is in the tens of byte I think, so I made the file in the test bigger to make it work.

Reply:
That sounds strange, considering that the test `rcreate`s a new repository in both cases.

Hmm, due to it using new key material, the chunker might behave differently, cutting different chunks, leading to slightly different compressed chunk sizes or different chunk counts.

Try replacing RK_ENCRYPTION with `"--encryption=none"` to avoid this.

Reply:
It also fails without encryption

Reply:
So, this is kind of resolved using the bigger test file now?